tests/test_class_basics.rs

```diff
@@ -148,7 +148,7 @@ struct EmptyClassInModule {}
 
 // Ignored because heap types do not show up as being in builtins, instead they
 // raise AttributeError:
-// https://github.com/python/cpython/blob/master/Objects/typeobject.c#L544-L573
+// https://github.com/python/cpython/blob/f9c5e3f5f61cd380f8a17c814766fc3730b7fbdf/Objects/typeobject.c#L544-L573
```

Comment:
Could we link a tag instead of a commit hash? Those should be stable as well but would give more context as to when this was valid.

Reply:
sure